web_ir_actions_act_window_none/static/src/js/web_ir_actions_act_window_none.js

```diff
@@ -0,0 +1,19 @@
+//-*- coding: utf-8 -*-
+//?? 2016 Therp BV <http://therp.nl>
+//License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+odoo.web_ir_actions_act_window_none = function(instance)
```

Comment:
this is not yet v9 api. https://www.odoo.com/documentation/9.0/reference/javascript.html#web-client should give you the idea